packages/template-retail-react-app/app/components/search/partials/suggestions.jsx

```diff
@@ -6,28 +6,61 @@
-import {Text, Button, Stack, Box} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {FormattedMessage} from 'react-intl'
+import {
+    Text,
+    Button,
+    Stack,
+    Box,
+    Flex,
+    Image,
+    useMultiStyleConfig
+} from '@salesforce/retail-react-app/app/components/shared/ui'
 
 const Suggestions = ({suggestions, closeAndNavigate}) => {
+    const styles = useMultiStyleConfig('SearchSuggestions')
+
     if (!suggestions) {
         return null
     }
     return (
-        <Stack spacing={0} data-testid="sf-suggestion">
-            <Box mx={'-16px'}>
+        <Stack {...styles.suggestionsContainer} data-testid="sf-suggestion">
+            <Box {...styles.suggestionsBox}>
                 {suggestions.map((suggestion, idx) => (
                     <Button
-                        width="full"
+                        {...styles.suggestionButton}
                         onMouseDown={() => closeAndNavigate(suggestion.link)}
-                        fontSize={'md'}
                         key={idx}
-                        marginTop={0}
-                        variant="menu-link"
                     >
-                        <Text
-                            fontWeight="400"
-                            dangerouslySetInnerHTML={{__html: suggestion.name}}
-                        />
+                        <Flex align="center">
+                            {/* Reserve space for image for all, but only render if present */}
+                            <Box {...styles.imageContainer}>
+                                {(suggestion.type === 'product' ||
+                                    suggestion.type === 'category') &&
+                                    suggestion.image && (
+                                        <Image
+                                            src={suggestion.image}
+                                            alt=""
+                                            {...styles.suggestionImage}
+                                        />
+                                    )}
+                            </Box>
+                            <Box {...styles.textContainer}>
+                                <Text {...styles.suggestionName}>{suggestion.name}</Text>
+                                {/* For categories, show parentCategoryName if present */}
+                                {suggestion.type === 'category' &&
+                                    suggestion.parentCategoryName && (
+                                        <Text {...styles.categoryParent}>
+                                            {' '}
+                                            <FormattedMessage
+                                                defaultMessage=" in "
+                                                id="search_suggestions.category.in_parent"
+                                            />
+                                            {' ' + suggestion.parentCategoryName}
```

Comment:
```suggestion
                                           <FormattedMessage
                                             defaultMessage=" in {parentCategory}"
                                             id="search_suggestions.category.in_parent"
                                             values=({parentCategory: suggestion.parentCategoryName})/>
```
When localizing messages, you don't want to concatenate strings because the syntax is not always correct in other languages.

Reply:
Ya I'll work w/ ux to see if there's some treatment that we can add in instead of the word " in "